code-challenges.test.js

```diff
@@ -63,13 +147,56 @@ var middleLetters1 = "alpha"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+describe('When getMiddle is called', () => {
+  it('returns the middle letter when provided an odd number', () => {
+    //Arrange
+    const stringArgument = 'alpha'
+
+    //Act
+    const actualResult = getMiddle(stringArgument)
+
+    //Assert
+    expect(actualResult).toEqual('p')
+  })
+  
+  it('returns the two middle letters when provided an odd number', () => {
+    //Arrange
+    const stringArgument = 'rhinOceros'
+
+    //Act
+    const actualResult = getMiddle(stringArgument)
 
+    //Assert
+    expect(actualResult).toEqual('oc')
+  })
+
+})
 
 
 
 // b) Create the function that makes the test pass.
 
 
+const getMiddle = (str) => {
+  let lower = str.toLowerCase()
+  let start = 0
+  let howMany = 0
```

Comment:
great use of a variable to indicate the number of characters to be returned. So much more legible!